GameServerLib/API/ApiEventManager.cs

```diff
@@ -37,6 +37,8 @@
 [OnLevelUp]
 [OnLevelUpSpell]
 [OnMiss]
+[OnFinishDash]
```

Comment:
Remove these and use `OnMoveEnd`, `OnMoveSuccess` and `OnMoveFailure`.
An `OnDash` is unnecessary as all units have MovementParameters which can be checked.

Reply:
Soo i should remove OnDash and OnFinishDash and create OnMoveEnd, OnMoveSuccess and OnMoveFailure events instead? And publish them from Move functions instead of Dash? Or keep OnFinishDash and just do OnMove events instead OnDash? Sorry but i don't understand.